openlibrary/macros/EditionNavBar.html

```diff
@@ -33,6 +33,8 @@
   </ul>
   $if not desktop_only:
     <span class="nav-arrow">
-      <img src="/static/images/nav-arrow.svg" role="img">
+      <button title="$_('Go to next section')">
+        <img src="/static/images/nav-arrow.svg" alt="">
+      </button>
     </span>
```

Comment:
Heads up: This was meant to replace the span with the button, and you'll have to update some the CSS as well